Source/Renderer/UniformState.js

```diff
@@ -846,6 +850,23 @@ define([
     }
 
     /**
+     * Synchronizes the frustum's state with the camera state.  This is called
+     * by the {@link Scene} when rendering to ensure that automatic GLSL uniforms
+     * are set to the right value.
+     *
+     * @param {Object} camera The camera to synchronize with.
+     */
+    UniformState.prototype.updateCamera = function(camera) {
```

Comment:
@bagnell so far is this inline with your vision of having separate viewports with separate cameras for things like picture-in-picture?